osidb/tests/test_flaw.py

```diff
@@ -1512,3 +1512,12 @@ def test_special_handling_modules(self):
 
         assert "special_handling_flaw_missing_summary" in flaw2._alerts
         assert "special_handling_flaw_missing_statement" in flaw2._alerts
+
+    def test_validate_private_source_no_ack(self, private_source, public_source):
```

Comment:
I would also add a test case for that twofold private-public source.

Reply:
Makes sense, added another test case.